### PR TITLE
Overhaul Azure egress IP article

### PR DESCRIPTION
**Pull request by marians**
Follow-up to https://github.com/giantswarm/docs/pull/1025

This PR aligns mainly capitalization and formatting with our general style. Two main guidelines:

- **Use sentence casing**. That means, don't use extra Uppercase spelling, except for the terms we use it for. No need to do this for resources that Azure or any cloud provider wants to brand, like `Public IP` or `EC2 Instance`.
- Use `code literals` only where technical terms are mentioned, e. g. a CRD name. No need to use it for terms like "workload cluster".

Other changes are explained via inline comments.

### CODE REVIEW

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: Egress IP address on Azure
-title: Egress IP address on Azure
-description: How to reuse an existing Public IP address for egress traffic of worker nodes on Azure.
+title: Setting an egress IP address on Azure
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:3]: Title can be descriptive and a bit longer

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+description: How to reuse an existing public IP address for outgoing traffic of worker nodes on Azure.
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:4]: Using different wording than "egress" here might help make it easier to understand.

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 3]
```diff
@@ -10,29 +10,28 @@ owner:
...
 user_questions:
-- How can I customize the public IP address for egress traffic?
+  - How can I customize the public IP address for egress traffic on Azure?
 ---
 
-# Egress IP address on Azure
+# Setting an egress IP address on Azure
 
-Giant Swarm's `Workload Clusters` use [Azure NAT Gateways](https://docs.microsoft.com/en-us/azure/virtual-network/nat-gateway/nat-overview?ocid=AID754288&wt.mc_id=azfr-c9-scottha,CFID0658) to provide internet access to node pool nodes.
+Giant Swarm's workload clusters on Azure use [NAT gateways](https://docs.microsoft.com/en-us/azure/virtual-network/nat-gateway/nat-overview) to provide internet access to worker nodes.
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:18]: Simplified the URL, as the parameters are not needed

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 4]
```diff
@@ -10,29 +10,28 @@ owner:
...
 
 In order to do so:
 
-1. Create a Public IPv4 Address in the same subscription the `Workload Cluster` will be created into. The IP Address can
-be in any `Resource Group`, but it needs to use the `Standard` SKU and be in the same Region as the `Workload Cluster`.
-2. Set the `Resource ID` of the Public IP Address as a value for the `giantswarm.io/workers-egress-external-public-ip`
-annotation in the `AzureCluster` `Custom Resource` as in the following example:
+1. Create a public IPv4 address in the same subscription the workload cluster will be created into. The IP Address can
+be in any resource group, but it needs to use the `Standard` SKU and be in the same region as the workload cluster.
+2. Edit the workload cluster's [`AzureCluster`]({{< relref "/ui-api/management-api/crd/azureclusters.infrastructure.cluster.x-k8s.io.md" >}}) resource via the Management API and set the resource ID of the public IP address as a value for the `giantswarm.io/workers-egress-external-public-ip` annotation, as in the following example:
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:34]: - Adding a link to the CRD reference
- Changing the sentence structure to make it easier to understand

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 5]
```diff
@@ -46,15 +45,15 @@ spec:
   ...
 ```
 
-You can set the annotation to either an existing `Workload Cluster` or a brand-new one.
+You can set the annotation to either an existing workload cluster or a brand-new one.
+
+To enable and disable the use of an external IP address for an existing workload cluster, just add or remove the annotation from the `AzureCluster` resource at any time. Reconciliation takes just a few minutes to switch between the two modes and there shouldn't be any significant network packet loss while switching.
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:50]: Moved this from limitations, as it's not a limitation,  but in fact a nice feature.

File: src/content/advanced/egress-ip-address-azure/index.md [hunk 6]
```diff
@@ -46,15 +45,15 @@ spec:
...
-  removing the `annotation` from the `AzureCluster` Custom Resource. Reconciliation takes just a few minutes to switch
-  between the two modes and there shouldn't be any significant network packet loss while switching.
+- You can't use the same public IP address for more than one workload cluster at the same time. This is an Azure limitation.
+
 - This feature is only supported for Node Pools and not for the Master node.
-- If you enable the feature in an existing `Workload Cluster`, the previously used `Public IP` will not be deleted. This can lead to some undesired increase in the Azure bill.
+- If you enable the feature in an existing workload cluster, the previously used public IP will not be deleted. This can lead to some undesired increase in the Azure bill.
   If the external IP address feature is disabled again, the automatically generated public IP address will be reused.
-- There is no frontend support for this feature. The only way of enabling it is by editing the `AzureCluster` Custom Resource via the `Management Cluster` Kubernetes API.
+- There is no user interface support for enabling or disabling this setting.
```
Comment (marians) [inline src/content/advanced/egress-ip-address-azure/index.md:59]: Simplified, as I thought it's clear enough how it can be done.

**Review by whites11 — APPROVED** [review]
Makes sense, thanks for the improvements Marian

**Pull request merged**